pkg/userdata/ubuntu/testdata/openstack-overwrite-cloud-config.golden

```diff
@@ -42,37 +28,47 @@ write_files:
     set -xeuo pipefail
 
     sysctl --system
-    mkdir -p /opt/bin
-    apt-key add /opt/docker.asc
-    apt-key add /opt/kubernetes.asc
     apt-get update
 
-    # Hetzner's Ubuntu Bionic comes with swap pre-configured, so we force it off.
+    # Make sure we always disable swap - Otherwise the kubelet won't start'.
     systemctl mask swap.target
     swapoff -a
 
-    # If something failed during package installation but one of docker/kubeadm/kubelet was already installed
-    # an apt-mark hold after the install won't do it, which is why we test here if the binaries exist and if
-    # yes put them on hold
-    set +e
-    which docker && apt-mark hold docker.io docker-ce
-    which kubelet && apt-mark hold kubelet
-    which kubeadm && apt-mark hold kubeadm
-
-    # When docker is started from within the apt installation it fails with a
-    # 'no sockets found via socket activation: make sure the service was started by systemd'
-    # Apparently the package is broken in a way that it gets started without its dependencies, manually starting
-    # it works fine thought
-    which docker && systemctl start docker
-    set -e
-    DEBIAN_FRONTEND=noninteractive apt-get -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" dist-upgrade -y
-    if [[ -e /var/run/reboot-required ]]; then
-      reboot
+    # crictl
+    if [ ! -f /usr/local/bin/crictl ]; then
+        
+        wget -O /opt/crictl.tar.gz https://github.com/kubernetes-sigs/cri-tools/releases/download/v1.11.1/crictl-v1.11.1-linux-amd64.tar.gz
```

Comment:
Please do the `download and extract` in one step via a pipe:

`curl -L --fail https://github.com/kubernetes-sigs/cri-tools/releases/download/v1.11.1/crictl-v1.11.1-linux-amd64.tar.gz|tar -xvz`